### DISCUSSION TITLE
AI

### DISCUSSION THREAD

**Discussion by anwarrahim** [community/community#121278] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to know AI weaknesses

**Comment by niladri-1** [comment, marked as answer]
Hi, @anwarrahim,
## During AI/ML larning, I noticed the weaknesses of AI

1. **Creativity**: AI lacks it.
2. **Data Dependency**: Relies heavily on data.
3. **Ethical Concerns**: Bias and fairness issues.
4. **Common Sense**: AI lacks it.
5. **Vulnerability**: Susceptible to manipulation.
6. **Uncertainty**: Struggles with it.
7. **Resource Intensive**: Requires a lot of computing power.
8. **Adaptability**: Difficulty in adapting.
9. **Explainability**: Decisions are often unclear.
10. **Job Displacement**: Raises employment concerns.

**Reply by anwarrahim** [reply]
Thanks 🤍

**Reply by niladri-1** [reply]
If u like this, give that mark as answed
